modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseSourceCheck.java

```diff
@@ -627,14 +639,20 @@ protected String getVariableName(String methodCall) {
 	}
 
 	protected String getVariableTypeName(
-		String content, String fileContent, String variableName) {
+		String content, String fileContent, String fileName, String variableName)
+		{
 
-		return getVariableTypeName(content, fileContent, variableName, false);
-	}
+			try {
+				return getVariableTypeName(content, fileContent, fileName, variableName, false);
+			}
```

Comment:
@TamyrisTorres, we shouldn't be adding `try...catch` in every place that we invoke the method `getVariableTypeName`, actually we should handle the exception inside the mentioned method using a single `try...catch` since it prevents code duplication and reduce code maintenance cost. Does it make sense to you?

By the way, at this moment, this exception it's too generic. What exception are you trying to catch?